modules/light-clients/06-solomachine/types/misbehaviour.go

```diff
@@ -10,48 +10,48 @@ import (
 	"github.com/cosmos/ibc-go/v3/modules/core/exported"
 )
 
-var _ exported.Misbehaviour = &Misbehaviour{}
+var _ exported.Misbehaviour = &DuplicateSignatures{}
 
 // ClientType is a Solo Machine light client.
-func (misbehaviour Misbehaviour) ClientType() string {
+func (ds DuplicateSignatures) ClientType() string {
 	return exported.Solomachine
 }
 
 // GetClientID returns the ID of the client that committed a misbehaviour.
-func (misbehaviour Misbehaviour) GetClientID() string {
-	return misbehaviour.ClientId
+func (ds DuplicateSignatures) GetClientID() string {
+	return ds.ClientId
 }
 
 // Type implements Evidence interface.
-func (misbehaviour Misbehaviour) Type() string {
+func (ds DuplicateSignatures) Type() string {
 	return exported.TypeClientMisbehaviour
 }
 
 // ValidateBasic implements Evidence interface.
```

Comment:
Should these godocs read `Misbehaviour` instead of `Evidence`?

Reply:
yes. The SDK x/evidence module was used once upon a time...

Reply:
Updated! :)